pipenv/exceptions.py

```diff
@@ -137,14 +137,14 @@ def show(self, file=None):
 
 
 class PipenvFileError(FileError):
-    formatted_message = "{} {{}} {{}}".format(click.style("ERROR:", fg="red", bold=True))
+    formatted_message = "{} {{}} {{}}".format("[bold red]ERROR:[/bold red]")
```

Comment:
I am not sure -- does this work or does it have to be `[bold][red]`?

Reply:
yes it works like that, it can be either

Reply:
Ah ok thanks --sorry I accidentally marked this as ready for review -- feel free to change back.